Makefile

```diff
@@ -45,18 +45,10 @@ lint:
 test:
 	CGO_ENABLED=1 go test -race ./...
 
-.PHONY: dep
-dep:
-	dep ensure -v
```

Comment:
I think we can replace this with `go mod download`

Reply:
Should we still call that target "dep"?

Reply:
yeah, SGTM, since it's from "dependency" so whatever 🤷‍♂️